### PR TITLE
Rename presto to trino

### PR DESCRIPTION
**Pull request by Praveen2112**

### CODE REVIEW

**Comment by findepi** [comment]
We probably need to migrate from Travis to GHA asap (first?)

File: trinodev/centos7-oj8/Dockerfile [hunk 1]
```diff
@@ -11,11 +11,11 @@
 # limitations under the License.
 
 FROM library/centos:7
-MAINTAINER Presto community <https://prestosql.io/community.html>
+MAINTAINER Trino community <https://trinodb.io/community.html>
 
 COPY ./files /
 
-# Install Java and presto-admin dependences
+# Install Java and trino-admin dependences
```
Comment (findepi) [inline trinodev/centos7-oj8/Dockerfile:18]: trino-admin does not exist yet. no change requested

File: trinodev/hdp2.6-hive/Dockerfile [hunk 2]
```diff
@@ -10,8 +10,8 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM prestodev/centos6-oj8:unlabelled
-MAINTAINER Presto community <https://prestosql.io/community.html>
+FROM trinodev/centos7-oj11:unlabelled
+MAINTAINER Trino community <https://trinodb.io/community.html>
```
Comment (findepi) [inline trinodev/hdp2.6-hive/Dockerfile:14]: http 404, use slack.html instead

**Review by findepi — COMMENTED** [review]
LGTM, but the build is failing.

File: trinodev/hdp2.6-hive/Dockerfile (hunk 2 above)
Comment (electrum) [reply trinodev/hdp2.6-hive/Dockerfile]: We should remove this from all the images as it’s deprecated. There is a replacement label to indicate the maintainer, but I don’t see any value in having it. I’d rather just remove it.

File: .travis.yml [hunk 3]
```diff
@@ -2,20 +2,20 @@ language: generic
...
-  - IMAGE_TO_BUILD=prestodev/centos6-oj8-openldap-referrals
-  - IMAGE_TO_BUILD=prestodev/centos6-presto-admin-tests-build
-  - IMAGE_TO_BUILD=prestodev/centos7-oj11
-  - IMAGE_TO_BUILD=prestodev/dns
-  - IMAGE_TO_BUILD=prestodev/kerberos TEST_COMMAND=bin/test_kerberos.sh
-  - IMAGE_TO_BUILD=prestodev/gpdb-6 IMAGE_TO_TEST=gpdb-6
-  - IMAGE_TO_BUILD=prestodev/hive3.1-hive IMAGE_TO_TEST=hive3.1-hive
+  - IMAGE_TO_BUILD=trinodev/hdp2.6-hive-kerberized IMAGE_TO_TEST=hdp2.6-hive
+  - IMAGE_TO_BUILD=trinodev/hdp2.6-hive-kerberized-2
+  - IMAGE_TO_BUILD=trinodev/hdp3.1-hive-kerberized IMAGE_TO_TEST=hdp3.1-hive
```
Comment (findepi) [inline .travis.yml:7]: `prestodev` was a Docker hub org name.
we're not going to use `trinodev` and the images on GHCR (github container registry) are prefixed with `testing/`, so i think the directory name should be `testing` too

cc @losipiuk

**Comment by Praveen2112** [comment]
trinodev is maintained as a environment variable and as a path in centos (not sure if we would need to have it as testing)

File: Makefile [hunk 4]
```diff
@@ -24,7 +24,7 @@ TEST_SH=bin/test.sh
 BUILDDIR=build
 DEPDIR=$(BUILDDIR)/depends
 FLAGDIR=$(BUILDDIR)/flags
-ORGDIR=prestodev
+ORGDIR=testing
```
Comment (ssheikin) [inline Makefile:27]: Seems this parameter can be inlined as `testing` is quite strange name for organisation.  
cc @findepi @losipiuk

File: .travis.yml [hunk 5]
```diff
@@ -2,20 +2,20 @@ language: generic
...
-  - IMAGE_TO_BUILD=prestodev/iop4.2-hive IMAGE_TO_TEST=iop4.2-hive
-  - IMAGE_TO_BUILD=prestodev/spark3.0-iceberg
-  - IMAGE_TO_BUILD=prestodev/centos6-oj8-openldap-referrals
-  - IMAGE_TO_BUILD=prestodev/centos6-presto-admin-tests-build
-  - IMAGE_TO_BUILD=prestodev/centos7-oj11
-  - IMAGE_TO_BUILD=prestodev/dns
-  - IMAGE_TO_BUILD=prestodev/kerberos TEST_COMMAND=bin/test_kerberos.sh
-  - IMAGE_TO_BUILD=prestodev/gpdb-6 IMAGE_TO_TEST=gpdb-6
-  - IMAGE_TO_BUILD=prestodev/hive3.1-hive IMAGE_TO_TEST=hive3.1-hive
+  - IMAGE_TO_BUILD=testing/hdp2.6-hive-kerberized IMAGE_TO_TEST=hdp2.6-hive
```
Comment (ssheikin) [inline .travis.yml:5]: is this file has to be deleted and this pr has to be based on https://github.com/trinodb/docker-images/pull/83 ?

File: .travis.yml (hunk 3 above)
Comment (findepi) [reply .travis.yml]: BTW sphinx image is not for testing, so it should be `build/`

File: Makefile (hunk 4 above)
Comment (findepi) [reply Makefile:27]: "prestodev" was an org in Docker Hub parlance.
in GHCR parlance the org is "trinodb", and the images are nested in arbitrarily deep directory structures... or there are no directories, just slashes in the names, how you prefer.
Thus, if ORG variable stays, it should have "trinodb" value.

File: .travis.yml (hunk 5 above)
Comment (Praveen2112) [reply .travis.yml:5]: It has to be on top of #83

File: prestodev/cdh5.12-hive-kerberized/Dockerfile [hunk 6]
```diff
@@ -11,7 +11,6 @@
 # limitations under the License.
 
 FROM prestodev/cdh5.12-hive:unlabelled
-MAINTAINER Presto community <https://prestosql.io/community.html>
```
Comment (losipiuk) [inline prestodev/cdh5.12-hive-kerberized/Dockerfile:14]: Why not update to trino?

Comment (losipiuk) [reply prestodev/cdh5.12-hive-kerberized/Dockerfile:14]: I personally do not care much - but maybe there was a reason to have it.

Comment (Praveen2112) [reply prestodev/cdh5.12-hive-kerberized/Dockerfile:14]: Reference : https://github.com/trinodb/docker-images/pull/82#discussion_r552048228

**Review by ssheikin — CHANGES_REQUESTED** [review]
It has to be on top of #83 https://github.com/trinodb/docker-images/pull/82/files#r553077636

**Review by ssheikin — CHANGES_REQUESTED** [review]
It has to be on top of #83

**Review by ssheikin — APPROVED** [review]

**Comment by kokosing** [comment]
@Praveen2112 There are plenty of automation failures.

File: prestodev/hive3.1-hive/Dockerfile [hunk 7]
```diff
@@ -1,5 +1,4 @@
 FROM prestodev/centos7-oj8:unlabelled
-MAINTAINER Presto community <https://prestosql.io/community.html>
 
 RUN yum install -y \
     mariadb-server \
```
Comment (kokosing) [inline prestodev/hive3.1-hive/Dockerfile:4]: There are some iop related files, please remove them.

File: Makefile (hunk 4 above)
Comment (kokosing) [reply Makefile:27]: There are two directories: build and testing. Does makefile is able to build sphinx image?

File: prestodev/hive3.1-hive/Dockerfile (hunk 7 above)
Comment (Praveen2112) [reply prestodev/hive3.1-hive/Dockerfile:4]: I searched it .. It looks like most of the files are removed.

**Comment by Praveen2112** [comment]
> There are two directories: build and testing. Does makefile is able to build sphinx image?

I guess we don't build sphinx image. Ref : https://github.com/trinodb/docker-images/pull/77

**Review by kokosing — APPROVED** [review]

File: prestodev/centos6-oj8-openldap-referrals/files/etc/openldap/setup/createReferrals.ldif [hunk 8]
```diff
@@ -1,15 +0,0 @@
-dn: ou=World,dc=presto,dc=testldap,dc=com
```
Comment (losipiuk) [inline prestodev/centos6-oj8-openldap-referrals/files/etc/openldap/setup/createReferrals.ldif:1]: is this one gone?

**Review by losipiuk — COMMENTED** [review]
I think all name for all images should be `ghcr.io/trinodb/testing/...` not just `testing/...`. This is how we push them to ghcr and how we reference them from test code in trino.

Another option would be to keep `testing/...` here and only tag as `ghcr.io/trinodb/testing` before release and push, but I think it would be more confusing.

@findepi WDYT?

File: prestodev/centos6-oj8-openldap-referrals/files/etc/openldap/setup/createReferrals.ldif (hunk 8 above)
Comment (Praveen2112) [reply prestodev/centos6-oj8-openldap-referrals/files/etc/openldap/setup/createReferrals.ldif:1]: The file is replaed - testing/centos6-oj8-openldap-referrals/files/etc/openldap/setup/createReferrals.ldif

**Comment by findepi** [comment]
> @findepi WDYT?

As long as it doesn't require us to have `ghcr.io/trinodb/testing` in the _file paths_, i like your proposal @losipiuk

**Comment by losipiuk** [comment]
> > @findepi WDYT?
> 
> As long as it doesn't require us to have `ghcr.io/trinodb/testing` in the _file paths_, i like your proposal @losipiuk

We discussed it offline with @Praveen2112 and for now let's keep the image names during build shorter, and we will add extra tagging to release process which does the pushing. This will be addressed in followup PR.

**Review by losipiuk — APPROVED** [review]

**Pull request merged**